xbmc/settings/Settings.cpp

```diff
@@ -456,6 +456,9 @@ bool CSettings::InitializeDefinitions()
 #elif defined(TARGET_ANDROID)
   if (CFile::Exists(SETTINGS_XML_FOLDER "android.xml") && !Initialize(SETTINGS_XML_FOLDER "android.xml"))
     CLog::Log(LOGFATAL, "Unable to load android-specific settings definitions");
+#elif defined(HAS_LIBAMCODEC)
```

Comment:
i can add aml-linux.xml and aml-android.xml if its wanted...

actually i dont have a special usecase, originally i tried using https://github.com/popcornmix/xbmc/commit/138abba9cf31a7879e5c01676d665e088c0ce199 on a aml box, but this dont work (yet). But if you look at https://github.com/popcornmix/xbmc/blob/138abba9cf31a7879e5c01676d665e088c0ce199/system/settings/rbp.xml maybe the one or other option could be usefull on (slow) AML boxes.